### PR TITLE
MGMT-16493: Run OPCT against cluster in OCI

### PR DESCRIPTION
**Pull request by adriengentil**

### CODE REVIEW

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

File: ci-operator/step-registry/assisted/oci/assisted-oci-workflow.yaml [hunk 1]
```diff
@@ -6,13 +6,17 @@ workflow:
...
-      - ref: assisted-baremetal-test
+      - ref: provider-certification-tool-run
+      - ref: provider-certification-tool-results
     post:
       - ref: assisted-common-gather
+      - ref: provider-certification-tool-destroy
       - ref: assisted-oci-teardown
     env:
-      TEST_SUITE: minimal
+      OPCT_DEV_EXTRA_CMD: --plugins-image=openshift-tests-provider-cert:devel
```
Comment (mtulio) [inline ci-operator/step-registry/assisted/oci/assisted-oci-workflow.yaml:20]:
```suggestion
      OPCT_DEV_EXTRA_CMD: --plugins-image=openshift-tests-provider-cert:v0.4.1
```

File: ci-operator/step-registry/assisted/oci/assisted-oci-workflow.yaml [hunk 2]
```diff
@@ -6,13 +6,17 @@ workflow:
...
       - ref: assisted-oci-setup
       - chain: assisted-common-pre
+      - ref: provider-certification-tool-conf-dedicated
+      - ref: provider-certification-tool-conf-setup
+      - ref: provider-certification-tool-wait-ready
     test:
-      - ref: assisted-baremetal-conf
-      - ref: assisted-baremetal-test
+      - ref: provider-certification-tool-run
+      - ref: provider-certification-tool-results
```
Comment (mtulio) [inline ci-operator/step-registry/assisted/oci/assisted-oci-workflow.yaml:14]: We may need a chain here =[

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 3]
```diff
@@ -886,6 +886,9 @@ tests:
   optional: true
   steps:
     cluster_profile: oci-assisted
+    env:
+      ASSISTED_CONFIG: |
+        NUM_WORKERS=3
     workflow: assisted-oci
```
Comment (mtulio) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:892]: @adriengentil looking the latest run[1], maybe we need a higher timeout for this job or workflow:
```suggestion
    workflow: assisted-oci
    timeout: 6h0m0s
```

[1] https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/48274/rehearse-48274-pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted/1755533740458643456#1:build-log.txt%3A64-65

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted-opct

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted-opct

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-oci-assisted-opct

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-opct-oci-assisted

**Pull request merged**